### DISCUSSION TITLE
Bug with Captain's Log Values

### DISCUSSION THREAD

**Discussion by pscotthill** [bcholmes/StarTrek2d20#169] [General]
I had manually created a captain using the Captain's Log rules.  I was THRILLED to see that doing this had been added to this app!

I decided to "recreate" my captain using the application.  While doing so, I clicked on the randomizer button a few times for Value in Step Two: Environment.  I did this a few times, then pasted the one I had created originally over the last random result.  When I exported the character to PDF, the last random result appeared on the character sheet instead of the one I pasted in.

**Comment by bcholmes** [comment]
Thanks for reporting this. It was a subtle bug.
